### PR TITLE
Updating flink service check to point to correct dashboard url

### PR DESCRIPTION
**Pull request by AshishKhatkar**
Fixing the flink service health check based on changes released in #2925

Changes tested on infrastage : https://fluffy.yelpcorp.com/i/WHmQ80fvg45tZczQB1nQK755VG6tHkn6.html

### CODE REVIEW

File: paasta_tools/check_flink_services_health.py [hunk 1]
```diff
@@ -122,6 +142,8 @@ def check_flink_service_health(
     num_healthy_jobmanagers = healthy_flink_containers_cnt(si_pods, "jobmanager")
     num_healthy_taskmanagers = healthy_flink_containers_cnt(si_pods, "taskmanager")
 
+    servie_cr_name = get_cr_name(si_pods)
```
Comment (JoeMalt) [inline paasta_tools/check_flink_services_health.py:145]: nit: typo in `service` here

File: paasta_tools/check_flink_services_health.py [hunk 2]
```diff
@@ -105,6 +108,23 @@ def check_under_registered_taskmanagers(
...
+        pod
+        for pod in si_pods
+        if pod.metadata.labels["flink.yelp.com/container-type"] == "jobmanager"
+        and is_pod_ready(pod)
+        and container_lifetime(pod).total_seconds() > 60
+    ]
+    if len(jobmanager_pod) == 1:
+        return jobmanager_pod[0].metadata.name.split("-jobmanager-")[0]
+    else:
+        return ""
```
Comment (JoeMalt) [inline paasta_tools/check_flink_services_health.py:125]: what do you think about erroring out here rather than returning an empty string?

Comment (AshishKhatkar) [reply paasta_tools/check_flink_services_health.py:125]: I handled this case in `check_under_registered_taskmanagers`.

File: paasta_tools/check_flink_services_health.py (hunk 1 above)
Comment (AshishKhatkar) [reply paasta_tools/check_flink_services_health.py:145]: done

File: paasta_tools/check_flink_services_health.py (hunk 2 above)
Comment (JoeMalt) [reply paasta_tools/check_flink_services_health.py:125]: Yeah I see it's handled there, I just wonder whether a more specific error message here is useful. Not a big deal though, don't block on this.

**Review by JoeMalt — APPROVED** [review]
lgtm

File: paasta_tools/check_flink_services_health.py (hunk 2 above)
Comment (AshishKhatkar) [reply paasta_tools/check_flink_services_health.py:125]: Yeah, my understanding is if the jobmanager is not running we won't have running dashboard which will be pointed out by the output message handling. Also, in such cases jobmanager pod check will also fail.

**Review by gvolco — APPROVED** [review]

**Pull request merged**